includes/vendor/myparcelnl/sdk/src/Model/Consignment/AbstractConsignment.php

```diff
@@ -698,8 +720,12 @@ public function setCity($city)
      * @return string|null
      * @var bool
      */
-    public function getStreet($useStreetAdditionalInfo = false)
+    public function getStreet($useStreetAdditionalInfo = false): ?string
```

Comment:
Als street null is, dan wil je toch een error geven? Wanneer kan je een consignment hebben zonder street? Kom maar even lang om hierover te babbelen. Het teruggeven van null heeft altijd gevaar omdat nu iedereen rekening mee moet houden dat er ook null terug kan komen.

Reply:
Done